rwengine/src/objects/CharacterObject.cpp

```diff
@@ -645,3 +645,9 @@ void CharacterObject::useItem(bool active, bool primary) {
         }
     }
 }
+
+void CharacterObject::clearInventory() {
+	for (int slot = 0; slot < kMaxInventorySlots; ++slot) {
```

Comment:
Range loop?

Reply:
in this case range loop make things complicated, because we are getting a value which we need to unpack

Reply:
You are right, I haven't noticed usage of slot.